src/sql/workbench/parts/connection/browser/advancedPropertiesController.ts

```diff
@@ -23,7 +23,7 @@ export class AdvancedPropertiesController {
 		this._options = this._advancedDialog.optionValues;
 	}
 
-	public showDialog(providerOptions: azdata.ConnectionOption[], container: HTMLElement, options: { [name: string]: any }): void {
```

Comment:
I found this debt during my investigation, the container wasn't being used, (nor should it).